Makefile

```diff
@@ -153,7 +153,7 @@ release: build-binaries build-docker-images push-docker-images generate-k8s-yaml
 
 release-windows: build-binaries-windows build-docker-images-windows push-docker-images-windows upload-resources-to-github-windows
 
-test: spellcheck shellcheck unit-test e2e-test compatibility-test license-test go-report-card-test helm-sync-test helm-version-sync-test helm-lint
+test: spellcheck shellcheck unit-test e2e-test compatibility-test license-test helm-sync-test helm-version-sync-test helm-lint
```

Comment:
add go-linter here?

Reply:
yeah, I can. Was debating if we should put it before the `go.mod` addition. It works on my machine though so i'll send!